velox/connectors/hive/PartitionIdGenerator.h

```diff
@@ -34,7 +34,8 @@ class PartitionIdGenerator {
       const RowTypePtr& inputType,
       std::vector<column_index_t> partitionChannels,
       uint32_t maxPartitions,
-      memory::MemoryPool* pool);
+      memory::MemoryPool* pool,
+      bool partitionPathAsLowerCase = true);
```

Comment:
Please update the function comment. And does the function param `partitionPathAsLowerCase` need a default value?

Reply:
@jinchengchenghh Removed the default value.